content/posts/euc/eso.md

```diff
@@ -0,0 +1,146 @@
+- Monitoring
+- Day2Ops
+lightgallery: true
+---
+{{<image src="/img/euc/eso/eso-round.png" caption="Edge Services Observability">}}
+<p class="cap">
+Unified Access Gateway (UAG) is a key component in most VMware End User Computing deployments, including Workspace ONE and Horizon. The UAG hosts various Edge Services like VMware Tunnel, Content Gateway, Secure Email Gateway, Reverse Proxy and Horizon Secure (External) Access. Additionally, the UAG can also provide DMZ Authentication and Identity Bridging. These services play a pivotal role in providing Zero Trust Network Access to corporate resources.
+</p>
+
+Now that we have established the significance of the UAGs, it is imperative that any organization would require metrics from monitoring the UAGs. While there are options to monitor the UAG from an appliance/infrastructure perspective, customers also require metrics at the Edge Service level (Eg: Authenticated VPN Tunnel Sessions etc.) to gain visibility into the connections being authorized into the company network to access resources. Currently, there is no centralized view of metrics consolidated from all UAGs across all Edge Services, nor is there a way to visualize trends over a period. __Project ESO – Edge Services Observability__ (released as [VMware Fling](https://flings.vmware.com/edge-services-observability "Project ESO")), is a monitoring solution that polls Edge Services metrics from all UAGs and renders the data as dashboards for visualization. Additionally, administrators have the ability to configure alerts based on conditions to get notified and detect anomalies.
+
+## Features
+ESO is delivered as a _ready-to-deploy_ virtual appliance, containing the following:
+ 1. Web based Admin Portal for configuring the list of the UAGs that need to be monitored
+ 2. Curated Dashboards with customizable widgets for visualizing metrics polled from UAGs for
+	- VMware Tunnel
+	- VMware Horizon
+> **Bonus:** Also included is a dashboard to monitor performance of the ESO appliance itself
+ 3. Ability to setup alerts based on configurable triggers
+
+## Deployment
+ 1. Download the OVA file from [VMware Flings Site](https://flings.vmware.com/edge-services-observability "VMware Flings")
+ 2. Use vCenter to “Deploy OVF Template”
+ 3. Customize the following properties as required:
+	- Network Properties
+	- Password for logging into the Appliance Operating System (SSH/ vCenter Console)
+	- Password for accessing Admin Portal and Dashboards Page
+	- SSH Access
+	- Debug Logging for appliance deployment
+	{{<image src="/img/euc/eso/eso_12.png" caption="ESO - OVA Deployment - Customize template">}}
+	{{<image src="/img/euc/eso/eso_11.png" caption="ESO - OVA Deployment - Customize template">}}
+
+## Configuration
+ 1. Once the appliance is deployed and powered on, launch a web browser to access Admin Portal by navigating to the FQDN (over HTTPS) of the appliance
+	{{<image src="/img/euc/eso/eso_7.png" caption="ESO Admin Portal Login">}}
```

Comment:
please clean up images that are not used

Reply:
Cleaned up unused images